packages/origin-ui-irec-core/src/components/detail-view/DeviceDetailView.tsx

```diff
@@ -0,0 +1,324 @@
+
+    // useEffect(() => {
+    //     const getDeviceOrg = async () => {
+    //         const organization = await backendClient.organizationClient.getPublic(selectedDevice.ownerId);
+    //         setSelectedDevOrg(organization);
+    //     }
+    // }, [selectedDevice])
+
+    if (!configuration || !organizationClient || !selectedDevice) {
+        return <Skeleton variant="rect" height={200} />;
+    }
+
+    let tooltip = '';
+
+    const selectedDeviceType = selectedDevice.deviceType;
+    let image = solar;
+
+    if (selectedDeviceType.startsWith('Wind')) {
+        image = wind;
+    } else if (selectedDeviceType.startsWith('Hydro-electric Head')) {
+        image = hydro;
+    } else if (selectedDeviceType.startsWith('Thermal')) {
+        image = iconThermal;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Adam Terpening' });
+    } else if (selectedDeviceType.startsWith('Solid')) {
+        image = iconSolid;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'ahmad' });
+    } else if (selectedDeviceType.startsWith('Liquid')) {
+        image = iconLiquid;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'BomSymbols' });
+    } else if (selectedDeviceType.startsWith('Gaseous')) {
+        image = iconGaseous;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Deadtype' });
+    } else if (selectedDeviceType.startsWith('Marine')) {
+        image = iconMarine;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Vectors Point' });
+    }
+
+    const data = [
+        [
+            {
+                label: t('device.properties.facilityName'),
+                data: selectedDevice?.name
+            },
+            {
+                label: t('device.properties.deviceOwner'),
+                data: selectedDevice?.registrantOrganization
+            },
+            {
+                label: t('device.properties.complianceRegistry'),
+                data: 'what is instead?' // selectedDevice.complianceRegistry
+            },
+            {
+                label: t('device.properties.otherGreenAttributes'),
+                data: 'what is instead?' // selectedDevice.otherGreenAttributes
+            }
+        ],
+        [
+            {
+                label: t('device.properties.deviceType'),
+                data: selectedDevice?.deviceType,
+                image,
+                rowspan: 2
+            },
+            {
+                label: t('device.properties.meterReadCertified'),
+                data: 'what is instead?', // EnergyFormatter.format(selectedDevice.meterStats?.certified ?? 0),
+                tip: EnergyFormatter.displayUnit
+            },
+            {
+                label: t('device.properties.meterReadToBeCertified'),
+                data: 'what is instead?', // EnergyFormatter.format(selectedDevice.meterStats?.uncertified ?? 0),
+                tip: EnergyFormatter.displayUnit
+            },
```

Comment:
In general, this is a calculated value of "uncertified - certified" but need to clarify what time period we want to show this. Leave it for now.